### DISCUSSION TITLE
Crossfade Function

### DISCUSSION THREAD

**Discussion by GTraux** [KBLiveSolutions/open.control#16] [Feature Requests & Ideas]
Would it be possible to have the crossfade A/B integrated as an action or encoder? This could be an interesting use for LOOPER. I use the looper in a parallel track with "B" and use the crossfade MIDI mapped to blend with the input with the original of "A". Works for me as a layering tool that gives me more control for mixing purposes during a live performance.

**Comment by ll-dasein-ll** [comment]
Hi -

I'm sure you've already thought of this so maybe there's something I'm not considering properly about how you want this to work, but could you not just set the encoder to send a MIDI CC value and then midi map that to the crossfader?

**Reply by KBLiveSolutions** [reply]
> Hi -
> 
> I'm sure you've already thought of this so maybe there's something I'm not considering properly about how you want this to work, but could you not just set the encoder to send a MIDI CC value and then midi map that to the crossfader?

This would work, but you'd need to do this every time you create a set, which is not the point of open·control! :-)
It's an obvious miss from me, and I'm already working on this. Expect a fix soon.

**Reply by ll-dasein-ll** [reply]
👍 of course!  I knew there would be something I hadn't taken into account 😊

**Comment by GTraux** [comment]
Great to know this can be implemented fast. Curious to experiment with
encoder or toggle for quick transitions between A/B.

But I'll try today with the regular MIDI CC. Just for fun... Thanks

**Comment by KBLiveSolutions** [comment]
Hello

Can you please download and install this version of the Remote Script:
https://github.com/KBLiveSolutions/open.control-remote_script/archive/refs/heads/custom-actions.zip

It contains the crossfader addition.
Please report any issue here.
Cheers
